types/spectest/tests/share/no_quorum.go

```diff
@@ -1,6 +1,24 @@
 package share
 
+import (
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+	"github.com/herumi/bls-eth-go-binary/bls"
+)
+
 // NoQuorum tests msg with < unique 2f+1 signers
-func NoQuorum() *SpecTest {
-	panic("implement")
+func NoQuorum() *ShareTest {
+	ks := testingutils.Testing4SharesSet()
+	share := testingutils.TestingShare(ks)
+
+	msg := testingutils.TestingCommitMultiSignerMessage([]*bls.SecretKey{ks.Shares[1], ks.Shares[2]}, []types.OperatorID{1, 2})
+
+	return &ShareTest{
+		Name:                     "no quorum",
+		Share:                    *share,
+		Message:                  *msg,
+		ExpectedHasPartialQuorum: true,
+		ExpectedHasQuorum:        false,
+		ExpectedFullCommittee:    false,
+	}
```

Comment:
The logic is pretty much the same as `has_partial_quroum`?
If so maybe we might as well delete this test and change the name of `has_partial_quorum` to account for 2 scenarios?

Having the same test for 2 different scenarios might also be an ok approach, but I think we have enough clutter in our tests anyhow

Reply:
Agree